docs/designs/045-node-validation.md

```
@@ -0,0 +1,445 @@
+    * **open:** do not run this individual test if the batch sizes are not satisfied and implicitly consider the test as successful.
+    * **wait:** wait for the next batch window to see if other validation requests can help it meet its minimum requirements. This policy will result in other tests in the same validation request being blocked.
+
+## Triggering Validation
+
+**Option 1: Node object as the validation trigger (not recommended)**
+
+A false or missing node status condition could be used to signal that a new or existing node needs to be validated. This approach would be similar to how the node readiness controller detects whether a given node is ready to accept workloads.
+
+We could define an arbitrary number of node status conditions which map to a given set of tests. However, unless we always validate all GPUs, clients will need an ability to pass impacted GPUs and if a node has been fully drained to the API. As a result, to support per-validation configuration rather than using the global defaults, we can leverage a node annotation similar to the structured quarantineHealthEvent annotation in the fault-quarantine module.
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)